contributors/guide/coding-conventions.md

```diff
@@ -16,119 +18,54 @@ description: |
 ## Code conventions
 
   - Bash
-
     - https://google.github.io/styleguide/shell.xml
-
-    - Ensure that build, release, test, and cluster-management scripts run on
-macOS
+    - Ensure that build, release, test, and cluster-management scripts run on macOS
 
   - Go
-
-    - [Go Code Review
-Comments](https://github.com/golang/go/wiki/CodeReviewComments)
-
+    - [Go Code Review Comments](https://github.com/golang/go/wiki/CodeReviewComments)
```

Comment:
Not a blocker, but an FYI

For most links we prefer reference style markdown, so this would like this:

```
    - [Go Code Review Comments]
... <end of logical block>

[Go Code Review Comments]: https://github.com/golang/go/wiki/CodeReviewComments
```

or if there were multiple "Go Code Review Comments" in a doc:
```
   - [Go Code Review Comments][mylink]
... <end of logical block>

[mylink]: https://github.com/golang/go/wiki/CodeReviewComments
```

It generally makes the flow of things easier when reviewing and publishing to Hugo


There are some other little things with links and their best practices can be found here:
https://github.com/kubernetes/community/blob/master/contributors/guide/style-guide.md#links